dipy/tracking/local/tissue_classifier.pyx

```diff
@@ -95,7 +95,78 @@ cdef class ThresholdTissueClassifier(TissueClassifier):
             return ENDPOINT
 
 
-cdef class ActTissueClassifier(TissueClassifier):
+cdef class ConstrainedTissueClassifier(TissueClassifier):
+    r"""
+    Abstract class that takes as input included and excluded tissue maps.
+    The 'include_map' defines when the streamline reached a 'valid' stopping
+    region (e.g. gray matter partial volume estimation (PVE) map) and the
```

Comment:
In some cases, `gray` is used, whereas `grey` is used in other cases. Should be uniform.

Reply:
fixed! thx.